namespace.sh

```diff
@@ -2,7 +2,10 @@
 # SPDX-FileCopyrightText: (c) 2024 Ring Zero Desenvolvimento de Software LTDA
 # SPDX-License-Identifier: GPL-2.0-only
 
-export LUAXDP_GW=enp0s1 # replace it with your gateway interface
+LUAXDP_GW=$(ip r | awk 'NR ==1 {print $5}')
+export LUAXDP_GW=${LUAXDP_GW:="enp0s1"} # replace it with your gateway interface
```

Comment:
this line sets `"enp0s1"` when `LUAXDP_GW` is unset, right? the problem I see, if I got this right, is, when trying to set my own device, I'd have to comment out the first line, edit the second; that would be a bit tangled, IMO.

I usually see things like:
```
[ -z "$LUAXDP_GW" ] && LUAXDP_GW=$(ip r | awk 'NR ==1 {print $5}')
```
this choice would allow one to invoke it like: `LUAXDP_GW=ethX ./namespace.sh`, without having to edit any files. what do you think?

Reply:
Changed at bfb82825f7d7a8faafc5161c12720cbfc8b86fd2